UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -217,6 +225,10 @@ void InitializeEnvironment()
                 }
                 catch
                 {
+                    Debug.Log($"" +
+                        $"Couldn't connect to trainer on port {port} using API version {k_ApiVersion}. " +
```

Comment:
Hope this is OK, I want some information to make sure we're using the same port as python, but also want to make it clear that this is expected sometimes (when doing inference).